### ISSUE TITLE
Telemetry to activelook AR glasses

### ISSUE THREAD

**Issue by Braveman78** [FrSkyRC/ETHOS-Feedback-Community#3429]
Is it possible to transmitt Bluetooth telemetry from Frsky X20 to Activelook AR glasses?

Activelook is open source. 

https://www.activelook.net/pages/for-developers#development-toolbox

They have made it for Jeti and Powerbox:

https://www.vspeak-modell.de/en/glasses-module

**Comment by bsongis-frsky** [comment]
Interesting!

**Comment by Braveman78** [comment]
> Interesting!

Could this be done with a LUA to select what telemetry you want to be displayed on the AvtiveLook glasses?

**Comment by bsongis** [comment]
Yes it is possible. I have implemented everything needed in the Bluetooth module already. I can display timers already in the glasses.
But I am still working on 1.6.1 and RF 3.0.1
So I will continue later when I can find time

**Comment by bsongis-frsky** [comment]
Feature implemented in ethos26, already available in the latest nightlies

**Comment by jasc76** [comment]
Any doc on how to use it? Flashed the nightly and the BT menu is renamed and I can select Active Look as BT function
Just got my hands on a used set :-)

**Comment by bsongis** [comment]
It's very new!

**Comment by bsongis** [comment]
<img width="800" height="480" alt="Image" src="https://github.com/user-attachments/assets/2765c2f6-2de3-49f3-bba0-b4d5c6b5bfc6" />

**Comment by bsongis** [comment]
<img width="800" height="480" alt="Image" src="https://github.com/user-attachments/assets/8b23bc5f-068f-4612-afdb-67640d03fede" />

**Comment by bsongis** [comment]
<img width="800" height="480" alt="Image" src="https://github.com/user-attachments/assets/9e95c768-ee20-4bc5-893c-475f98f66caf" />

**Comment by bsongis** [comment]
And the rest is easy

**Comment by jasc76** [comment]
> And the rest is easy

it is 🥳👌😇

**Comment by mariuszlabedzki** [comment]
@bsongis is there any delay ? I would like to present the throttle percent and I'm wondering if there is noticeable lag between the throttle movement and the value presented in glasess

**Comment by robthomson** [comment]
Not from what, I have seen with my tests on rotorflight

**Comment by mariuszlabedzki** [comment]
> Not from what, I have seen with my tests on rotorflight

thank you !

**Comment by NorbertHeli** [comment]
I'm glad that engo 2 is supported as a telemetry output.

Would it be possible to have 5 or 6 output fields?

Or is the Bluetooth transmission to the goggles too slow? 
I could do without the time and transmitter battery.
Maybe like this?

<img width="725" height="435" alt="Image" src="https://github.com/user-attachments/assets/5e7e95d5-6cab-40eb-b34b-a0553b78c86d" />

**Comment by jasc76** [comment]
I doubt it will be readable going smaller

**Comment by NorbertHeli** [comment]
maybe, but you can select what you want
